pkgs/mime/tool/mime_types.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:mime/mime.dart';
+import 'package:mime/src/default_extension_map.dart';
+
+void main(List<String> args) {
```

Comment:
Add a comment saying what the script does. Fx:
```dart
/// Script to update the `../doc/media_types.md` file from the [defaultExtensionMap].
```

Doesn't have to be on `main`, but it's as good a place as any, and I know which scope I can refer to.